style/preview.less

```diff
@@ -143,4 +143,18 @@
             }
         }
     }
+
+    &.mobile {
+        width: 100%;
+    }
+}
+
+.wp-dark-screen {
+    background-color: rgba(0,0,0, 0.7);
+    z-index: 100; // TODO make it a variable?
```

Comment:
What do you think about adding a LESS variable for the z-index of the popup? That way here we could set the z-index of the screen to `@popup-z-index - 10` or something